website/docs/visual-testing/reporter.md

```diff
@@ -0,0 +1,103 @@
+---
+id: visual-reporter
+title: Visual Reporter
+---
+
+The Visual Reporter is a new feature introduced in the `@wdio/visual-service`, starting from version [v5.2.0](https://github.com/webdriverio/visual-testing/releases/tag/%40wdio%2Fvisual-service%405.2.0). This reporter allows users to visualize the JSON diff reports generated by the Visual Testing service and transform them into a human-readable format. It helps teams better analyze and manage the visual testing results by providing a graphical interface for reviewing the output.
+
+To utilize this feature, ensure you have the required configuration to generate the necessary `output.json` file. This document will guide you through setting up, running, and understanding the Visual Reporter.
+
+# Prerequisites
+
+Before using the Visual Reporter, make sure you have configured the Visual Testing service to generate JSON report files:
+
+```ts
+export const config = {
+    // ...
+    services: [
+        [
+            "visual-regression",
+            {
+                createJsonReportFiles: true, // Generates the output.json file
+            },
+        ],
+    ],
+};
+```
+
+For more detailed setup instructions, refer to the WebdriverIO [Visual Testing Documentation](./)
+
+# Installation
+
+To install the Visual Reporter, add it as a development dependency to your project using npm:
+
+```bash
+npm install @wdio/visual-reporter --save-dev
+```
+
+This will ensure the necessary files are available to generate reports from your visual tests.
+
+# Usage
+
+## Building the Visual Report
+
+Once you have run your Visual tests and they generated the `output.json` file, you can build the visual report using either the CLI or interactive prompts.
+
+### CLI Usage
+
+You can use the CLI command to generate the report by running:
+
+```bash
+npx @wdio/visual-reporter --jsonOutput=<path-to-output.json> --reportFolder=<path-to-store-report> [--logLevel=debug]
```

Comment:
seems like an unusual name for a CLI tool, we could also name it `npx wdio-report ...`, wdyt?

Reply:
Thanks for the feedback. I will call it `wdio-visual-reporter`, the `visual-reporter` name is already used and if I call that it will install that package.